jobrunner/sync.py

```diff
@@ -66,13 +67,23 @@ def api_post(*args, **kwargs):
     return api_request("post", *args, **kwargs)
 
 
-def api_request(method, path, *args, **kwargs):
+def api_request(method, path, *args, headers=None, **kwargs):
```

Comment:
What's the `headers` arg for? We always override it anyway it looks like.

Reply:
ah, I must have left that change in the stash

fixed, thanks